data/json/items/magazines.json

```diff
@@ -1,5 +1,21 @@
 [
     {   "type" : "MAGAZINE",
+        "id" : "garandclip",
+        "price" : 600,
+        "name" : "M1 Garand Clip",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A light-weight en-bloc clip for use with the M1 garand that will be ejected with a distinctive ping sound after the last round is fired.",
```

Comment:
If I recall, amusing discussion has occurred in the past regarding the noises an M1 would make. In a nutstall, while the pinging is due to the en-bloc clip landing on a hard surface, there's also a hollow ringing of sorts as the clip is ejected. o3o